generator/csharp.py

```diff
@@ -561,9 +561,11 @@ def tag_handler(tag, value):
                 
                 route_name = self._public_route_name(route_name, version)
                 auth_type = self._route_auth_map[(ns_name, route_name)]
-
-                return ('<see cref="{0}.{1}.Routes.{1}{2}Routes.{3}Async" />'.format(
-                        self._namespace_name, ns_name, auth_type, route_name))
+                links = []
```

Comment:
That's a good catch. I feel a lot fo SDK doesn't handle this multi-auth in docstring reference well.